services/reference/ipfs/index.md

```diff
@@ -0,0 +1,27 @@
+---
+description: Gas network documentation
```

Comment:
Missing the network description

Reply:
I meant to say `description` is incorrect, and the topic requires a description for what IPFS is/does etc.